rclcpp/src/rclcpp/executor.cpp

```diff
@@ -529,38 +529,11 @@ Executor::get_group_by_timer(rclcpp::TimerBase::SharedPtr timer)
   return rclcpp::callback_group::CallbackGroup::SharedPtr();
 }
 
-void
-Executor::get_next_timer(AnyExecutable & any_exec)
-{
-  for (auto & weak_node : weak_nodes_) {
-    auto node = weak_node.lock();
-    if (!node) {
-      continue;
-    }
-    for (auto & weak_group : node->get_callback_groups()) {
-      auto group = weak_group.lock();
-      if (!group || !group->can_be_taken_from().load()) {
-        continue;
-      }
-      auto timer_ref = group->find_timer_ptrs_if(
-        [](const rclcpp::TimerBase::SharedPtr & timer) -> bool {
-          return timer->is_ready();
```

Comment:
isn't necessary to check `is_ready` somewhere? As it was done before.

Reply:
No, it is taken care by rcl_wait set.

Reply:
Yes, I guessed that that was the case.
I don't understand why it was done differently before.